controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1472,6 +1492,17 @@ int cli_bml::get_device_operational_radios(const std::string &al_mac)
     return 0;
 }
 
+//================================================
+//New function to handle the new BML message
+int cli_bml::oren_print(const std::string &str)
+{
+    LOG(TRACE) << "=========> cli_bml::oren_print(): " << str << std::endl;
+    std::cout << "oren_print(): " << str << std::endl;
+    return 0;
```

Comment:
Its expected that  bml  return value type will be printed , see `printBmlReturnVals()`

Reply:
fixed